src/schema/annotation.yaml

```diff
@@ -81,48 +81,13 @@ classes:
     is_a: FunctionalAnnotationTerm
     description: >-
       A pathway is a sequence of steps/reactions carried out by an organism or community of organisms
-    slots:
-      - has_part
```

Comment:
Does it make sense for FunctionalAnnotationTerm to have no slots now???

Reply:
Good question. `Pathway` and `OrthologyGroup` `is_a: FunctionalAnnotationTerm`

This SPARQL query looks for things in our metadata that instantiate any subClass of `FunctionalAnnotationTerm`. It doesn't find anything.

```sparql
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX nmdc: <https://w3id.org/nmdc/>
select * where {
    graph nmdc:nmdc {
        ?subclass rdfs:subClassOf* nmdc:FunctionalAnnotationTerm
    }
    ?instance a ?subclass .
} 
```

You could argue that
- we tear **all** of this out, since we haven't used it
- we keep it but possible assert that `FunctionalAnnotationTerm` is `abstract` if we don't think that we would ever instantiate it directly.

Reply:
Curious about @cmungall 's thoughts since it was agreed to remove & this now also seems un-used?

Reply:
I'm not comfortable removing `FunctionalAnnotationTerm` just for the sake of tidying up after removing `Reaction`. That would mean removing `Pathway` and `OrthologyGroup`. Those haven't been instantiated in the MongoDB collection that I know how to navigate, but I'm pretty sure that they are searchable concepts int eh Data Portal.

Reply:
I added `todos` like

> todos:
>   - is Pathway instantiated in an MongoDB collection? Aren't Pathways searchable in the Data Portal?

Reply:
But to directly answer @mslarae13's question: this class is associated with the slots that it inherits form `OntologyClass` up to `NamedThing`. You could argue that it should be an `abstract` class since it doesn't declare it's own slots, and **may** not have been directly instantiated in MongoDB.